src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -213,6 +233,7 @@ internal static Conversion GetTrivialConversion(ConversionKind kind)
         internal static Conversion ImplicitDynamic => new Conversion(ConversionKind.ImplicitDynamic);
         internal static Conversion ExplicitDynamic => new Conversion(ConversionKind.ExplicitDynamic);
         internal static Conversion InterpolatedString => new Conversion(ConversionKind.InterpolatedString);
+        internal static Conversion Deconstruction => new Conversion(ConversionKind.Deconstruction);
```

Comment:
I use this one for error cases. Any time `MakeDeconstructionConversion` runs into errors, it will return this trivial conversion instance. Maybe I should rename it to `DeconstructionFailed`?